api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -160,4 +173,9 @@ message GrpcJsonTranscoder {
   //  the ``google/rpc/error_details.proto`` should be included in the configured
   //  :ref:`proto descriptor set <config_grpc_json_generate_proto_descriptor_set>`.
   bool convert_grpc_status = 9;
+
+  // URL unescaping policy. If this setting is not specified, the
```

Comment:
Could you also add comment: 
// this spec is only applied when extracting variable with multiple segments. 
// e.g. /foo/{x=*}/bar/{y=**}.   x variable is segment and y is multiple segments.
// for a path with /foo/abc/bar/aaa/bbb/ccc,  x=abc, y=aaa/bbb/ccc
maybe add examples with reserved characters and the extracting result for each spec.